fs/layer/node.go

```diff
@@ -160,6 +189,16 @@ func (n *node) logOperation(ctx context.Context, operationName string) {
 	}
 }
 
+func (n *node) incOperationFailureMetric(ctx context.Context, operationName string, errno syscall.Errno) {
+	if errno != 0 && !n.fs.noPrometheus {
```

Comment:
Is `!n.fs.noPrometheus` necessary?

Reply:
`logFSOperations bool` gives me an impression that metrics should only be released when `!noPrometheus` but it seems not the case :) I'll remove it.

Reply:
fixed